src/containers/FFFrontPage/FFFrontPage.jsx

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
Cries in .tsx :'(

Reply:
Skriv gjerne om slik at nye filer er typescript.

Reply:
Eller har vi ikkje typescript i ndla-frontend?

Reply:
Vi har ikke typescript i ndla-frontend så orket ikke å sette det opp.